gems/sorbet/lib/create-config.rb

```diff
@@ -11,14 +11,17 @@ class Sorbet::Private::CreateConfig
   SORBET_DIR = 'sorbet'
   SORBET_CONFIG_FILE = "#{SORBET_DIR}/config"
 
+  extend Sorbet::Private::UserInput
   include Sorbet::Private::StepInterface
 
   def self.main
     FileUtils.mkdir_p(SORBET_DIR)
 
-    File.open(SORBET_CONFIG_FILE, 'w') do |f|
-      f.puts('--dir')
-      f.puts('.')
+    if File.exist?(SORBET_CONFIG_FILE) && prompt_for_confirmation!("#{SORBET_CONFIG_FILE} already exists. Overwrite?")
```

Comment:
Should it prompt about the whole sir or just the config file?

Reply:
I don't think we need to ask permission to `mkdir -p`, since it can't overwrite anything users already had there.